recipes/libblitz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.10" %}
+
+package:
+  name: libblitz
+  version: {{ version }}
+
+source:
+  fn: blitz-{{ version }}.tar.gz
+  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
+  md5: 66268b92bda923735f2e3afc87dcb58a
+
+build:
+  number: 0
+  skip: true   # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+
+test:
+  commands:
+    - test -d $PREFIX/include/blitz  # [unix]
+
+about:
+  home: https://sourceforge.net/projects/blitz
+  license: LGPL 3
+  summary: A C++ class library for scientific computing which provides performance on par with Fortran 77/90
+
+extra:
+  recipe-maintainers:
+    - 183amir
+    - bioidiap
```

Comment:
@ocefpaf @pelson, thoughts? That does seem harmless (maybe a little noisy). I don't have strong feelings on it. Do either of you?